src/views/view_game.py

```diff
@@ -42,36 +124,41 @@ def init_window(self):
         pygame.display.flip()
 
         # Init sprites
-        all_sprites = pygame.sprite.RenderUpdates()
+        # all_sprites = pygame.sprite.Group()
+        #
+        # # Update the scene
+        # dirty = all_sprites.draw(self.window)
+        # pygame.display.update(dirty)
+        self.card_area_organizer.addCard(Card("KING"), "dealer")
+        self.card_area_organizer.addCard(Card("QUEEN"), "dealer")
 
-        # Update the scene
-        dirty = all_sprites.draw(self.window)
-        pygame.display.update(dirty)
+        self.card_area_organizer.addCard(Card("ACE", "SPADES"), "player", 0)
 
     def init_game_btns(self):
         # Display game buttons area
-        cfg_btns = self.view_config['game_buttons']
-        pygame.draw.rect(self.window, cfg_btns['color'],
-                         (cfg_btns['x'], cfg_btns['y'],
-                          cfg_btns['width'], cfg_btns['height']))
+        cfg_btns = game_view_config.game_buttons
+        pygame.draw.rect(self.window, cfg_btns.color,
+                         (cfg_btns.x, cfg_btns.y,
+                          cfg_btns.width, cfg_btns.height))
 
         self.quit_btn = Button(pos=(1100, 610),
-                          width=80,
-                          height=80,
-                          text='Quit',
-                          background=(180, 180, 180))
+                               width=80,
+                               height=80,
+                               text='Quit',
+                               background=(180, 180, 180))
         self.quit_btn.display(self.window)
 
     def add_card(self, card, area_name):
```

Comment:
Mmh, we already use this or just with the organizer ?

Reply:
Forgot to remove it, now we will only add_card *via* the organizer